pkg/appfile/appfile.go

```diff
@@ -329,7 +329,7 @@ func (af *Appfile) generateAndFilterCommonLabels(compName string) map[string]str
 		oam.LabelAppComponent: compName,
 	}
 	// merge application's all labels
-	finalLabels := util.MergeMapOverrideWithDst(Labels, af.AppLabels)
+	finalLabels := util.MergeMapOverrideWithDst(af.AppLabels, Labels)
```

Comment:
why we should exchange the order?

Reply:
The resources of one child application (identified as B) should carry `app.oam.dev/name: B` label.

However since the child application (B) is created and managed by the parent application (identified as A), B itself carries `app.oam.dev/name: A` label.

The KubeVela controller passes the labels of the child application B to its resources, but should not override `app.oam.dev/name: B`.

The logic here previously let `app.oam.dev/name: A` (the label passed from the child application B) overrides the label `app.oam.dev/name: B` (the label that indicates the owner of the resource). By changing the order here, we let the latter one to have higher priority.